tests/delta.test.js

```diff
@@ -18,13 +19,14 @@ const schema = new Schema({
  * @param {Y.XmlFragment} ytype
  * @param {Y.AbstractAttributionManager} [attributionManager]
  */
-const createProsemirrorView = (ytype, attributionManager) => {
-  const view = new EditorView(null, {
+const createProsemirrorView = async (ytype, attributionManager) => {
```

Comment:
Why is this async? 

Async should be avoided in editor bindings by all means.

Reply:
[We are waiting a tick](https://github.com/yjs/y-prosemirror/blob/8012bb7d9a3cd3545d79e8962d0f4056366ca583/src/index.js#L406-L408) before actually initializing the plugin, I can be convinced against it. But, this is useful for cases where prosemirror editor is created and immediately destroyed, (e.g. React's StrictMode), I've seen bugs because of this.